demo/tests/index.html

```diff
@@ -48,7 +48,7 @@
     <script src="Chart.js/Chart.js"></script>
     <script src="angular-chart.js/dist/angular-chart.js"></script>
     <script src="v-accordion/dist/v-accordion.js"></script>
-    <script src="satellizer/dist/satellizer.js"></script>
```

Comment:
check if you updated satelizer to the new version -> make bower install before dist !!!